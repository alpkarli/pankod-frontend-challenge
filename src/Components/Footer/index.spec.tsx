import { create } from "react-test-renderer";
import { Footer } from "@Components";

describe("Footer", () => {
    it("should match snapshot", () => {
        const component = create(<Footer />);
        expect(component.toJSON()).toMatchSnapshot();
    })
});