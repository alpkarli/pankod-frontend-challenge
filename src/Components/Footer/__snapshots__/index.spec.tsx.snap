// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot 1`] = `
<div
  className="footer"
>
  <div
    className="content-container"
  >
    <div
      className="footer__content"
    >
      <div
        className="footer__content--text"
      >
        Home |  Terms and Conditions |  Privacy Policy |  Collection Statement |  Help |  Manage Account
      </div>
      <div>
        <p>
          Copyright © 2016 DEMO Streaming. All Rights Reserved.
        </p>
      </div>
      <div
        className="footer__content--logo"
      >
        <div
          className="footer__content--social"
        >
          <img
            className="social"
            src="/images/social/facebook-white.svg"
          />
          <img
            className="social"
            src="/images/social/twitter-white.svg"
          />
          <img
            className="social"
            src="/images/social/instagram-white.svg"
          />
        </div>
        <div
          className="footer__content--store"
        >
          <img
            className="store"
            src="/images/store/app-store.svg"
          />
          <img
            className="store"
            src="/images/store/play-store.svg"
          />
          <img
            className="store"
            src="/images/store/windows-store.svg"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
